### PR TITLE
ADD no service connection error message

### PR DESCRIPTION
**Pull request by JeremiahUy**

### CODE REVIEW

File: src/components/ResultWrapper/ResultWrapper.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import AnalysisWrapper from '../AnalysisWrapper/AnalysisWrapper';
+
+const ResultWrapper = (props) => {
+  const { response, loadingAnalysis, event } = props;
+
+  const renderAction = (action) => {
+    /* if (action === 'anonymize' && response.anonymizeResult) {
```
Comment (JulianSagen) [inline src/components/ResultWrapper/ResultWrapper.jsx]: Would be cleaner if this comment where removed

**Review by JulianSagen — CHANGES_REQUESTED** [review]
All clean and good apart from the commented code in result wrapper

File: src/components/ResultWrapper/ResultWrapper.jsx (hunk 1 above)
Comment (JeremiahUy) [reply src/components/ResultWrapper/ResultWrapper.jsx]: will include a anonymize function soon, so this comment will be used, but ok

**Review by JulianSagen — APPROVED** [review]
Nice and clean 👍

**Pull request merged**